django/contrib/admin/static/admin/img/calendar-icons.svg

```diff
@@ -2,62 +2,40 @@
 <svg
    width="15"
    height="30"
-   viewBox="0 0 1792 3584"
+   viewBox="0 0 512 1024"
    version="1.1"
    id="svg5"
-   sodipodi:docname="calendar-icons.svg"
-   inkscape:version="1.3.2 (091e20ef0f, 2023-11-25, custom)"
-   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
-   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
    xmlns:xlink="http://www.w3.org/1999/xlink"
-   xmlns="http://www.w3.org/2000/svg"
-   xmlns:svg="http://www.w3.org/2000/svg">
-  <sodipodi:namedview
-     id="namedview5"
-     pagecolor="#ffffff"
-     bordercolor="#666666"
-     borderopacity="1.0"
-     inkscape:showpageshadow="2"
-     inkscape:pageopacity="0.0"
-     inkscape:pagecheckerboard="0"
-     inkscape:deskcolor="#d1d1d1"
-     showgrid="false"
-     inkscape:zoom="13.3"
-     inkscape:cx="15.526316"
-     inkscape:cy="20.977444"
-     inkscape:window-width="1920"
-     inkscape:window-height="1011"
-     inkscape:window-x="0"
-     inkscape:window-y="0"
-     inkscape:window-maximized="1"
-     inkscape:current-layer="svg5" />
-  <defs
-     id="defs2">
-    <g
-       id="previous">
+   xmlns="http://www.w3.org/2000/svg">
+   <!-- 
+   All icons are provided by Font Awesome Free, version 6.7.1 
+   Website: https://fontawesome.com 
+   License: Creative Commons Attribution 4.0 International (CC BY 4.0)
+   For more details, visit: https://fontawesome.com/license/free 
+   Copyright 2024 Fonticons, Inc.
+   -->
```

Comment:
Personally I’d find it easier if we kept the original licensing comments from FontAwesome as-is.